src/Iris/Env.hs

```diff
@@ -1,4 +1,6 @@
 {-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE RecordWildCards #-}
```

Comment:
`RecordWildCards` is enabled by default in `iris.cabal`, so you can remove this line